### ISSUE TITLE
The app requests a CA while I already have the CA chain embedded in the keystore

### ISSUE THREAD

**Issue by kghost** [schwabe/ics-openvpn#1391]
Using 0.7.25

My imported PKCS12 certificate includes the full/ CA chain

![Screenshot_20211008-015844_Settings](https://user-images.githubusercontent.com/154702/136439040-0558c547-fa47-4d81-a874-6b65cfe73e42.png)

But I got this error when connecting

![Screenshot_20211008-020137_OpenVPN_for_Android](https://user-images.githubusercontent.com/154702/136439132-405427db-e4a4-41bd-a767-6b357f4e84a0.png)

It works after manually extract the root CA from the PKCS12 file and set it into CA field.

BTW, my config file uses an embedded pkcs12 file like this:

```
#openvpn configurations

<pkcs12>
Base64 of pkcs12 file
</pkcs12>
```

**Comment by sevenrock** [comment]
Same error here, needed to revert to 0.7.24

**Comment by schwabe** [comment]
Already fixed in newer versions

**Comment by sevenrock** [comment]
ftr: yes, the bug is fixed, but not in the version 0.7.27 available on the play store, instead you need to install version 0.7.28pre available from http://plai.de/android/
Thanks for this great app!
